modules/heat_transfer/src/materials/HeatConductionMaterial.C

```diff
@@ -113,7 +115,12 @@ HeatConductionMaterialTempl<is_ad>::computeQpProperties()
   }
 
   if (_specific_heat_temperature_function)
+  {
     _specific_heat[_qp] = _specific_heat_temperature_function->value(qp_temperature);
+    if (_specific_heat_dT)
+      (*_specific_heat_dT)[_qp] = _specific_heat_temperature_function->timeDerivative(
+          MetaPhysicL::raw_value(qp_temperature));
```

Comment:
@dschwen the `DerivativeMaterialInterface` eliminates the need for this kind of thing right?